<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtArcWidgetPanel</class>
 <widget class="QWidget" name="qtArcWidgetPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QWidget" name="SelectPointsWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="StartPointButton">
          <property name="text">
           <string>Pick Start Point</string>
          </property>
          <property name="icon">
           <iconset resource="qtWidgetsIcons.qrc">
            <normaloff>:/icons/pqwidgets/edgepoint.png</normaloff>:/icons/pqwidgets/edgepoint.png</iconset>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="EndPointButton">
          <property name="text">
           <string>Pick End Point</string>
          </property>
          <property name="icon">
           <iconset resource="qtWidgetsIcons.qrc">
            <normaloff>:/icons/pqwidgets/edgepoint.png</normaloff>:/icons/pqwidgets/edgepoint.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="WholeArcButton">
          <property name="text">
           <string>Select Whole Arc</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLabel" name="StartPointLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Start Coordinate</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="EndPointLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>End Coordinate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="OperationsWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="CommonOperations">
           <item row="0" column="0" colspan="2">
            <widget class="QPushButton" name="EditButton">
             <property name="text">
              <string>Edit</string>
             </property>
             <property name="icon">
              <iconset resource="qtWidgetsIcons.qrc">
               <normaloff>:/icons/pqwidgets/contour.png</normaloff>:/icons/pqwidgets/contour.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QPushButton" name="ArcStraightenButton">
             <property name="text">
              <string>Make Straight Line</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="SubArcOpWidget" native="true">
           <layout class="QGridLayout" name="gridLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout" name="SubArcOperations">
              <item row="0" column="0">
               <widget class="QPushButton" name="ArcCollapseButton">
                <property name="text">
                 <string>Collapse (End Point to Start Point)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="MakeArcButton">
                <property name="text">
                 <string>Make Arc</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="WholeArcOpWidget" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout" name="WholeArcOperations"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="PickButtonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Close</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="ArcEditWidgtet" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="ContourLayout"/>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="EditButtonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="FinalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qtWidgetsIcons.qrc"/>
 </resources>
 <connections/>
</ui>
